workflows/raredisease.nf

```diff
@@ -295,6 +296,27 @@ workflow RAREDISEASE {
     )
     ch_versions = ch_versions.mix(CALL_REPEAT_EXPANSIONS.out.versions)
 
+    // STEP 1.7: SMNCOPYNUMBERCALLER
+    ALIGN.out.bam_bai
```

Comment:
You could for consistency use `ch_mapped.bam_bai` here instead of `ALIGN.out.bam_bai`

Reply:
You also have the option of using the bam and bai channels directly instead of splitting and combining the bam_bai channel. But I haven't checked if that would simplify the code. It might just end up looking exactly the same 😆

Reply:
> 

Tried it now, it ended up the same length, so I think I'll keep it like this to mimic what we do at glnexus and deepvariant.